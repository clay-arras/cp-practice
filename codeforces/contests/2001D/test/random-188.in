1
25
15 4 24 8 13 19 17 3 14 11 9 8 4 3 25 10 25 18 18 20 3 22 10 1 8
