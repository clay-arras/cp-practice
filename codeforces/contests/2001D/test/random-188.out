17
15 4 24 8 13 19 17 3 14 11 9 10 25 18 20 22 1
